STM32Cube/Tasks/trajectory.h

```diff
@@ -8,6 +8,54 @@
 #ifndef TRAJECTORY_H_
 #define TRAJECTORY_H_
 
+#include "FreeRTOS.h"
+#include "queue.h"
+
+extern xQueueHandle altQueue;
+extern xQueueHandle angleQueue;
+extern xQueueHandle extQueue;
+extern xQueueHandle apogeeQueue;
+
+float rocket_area(float extension);
```

Comment:
you should not actually declare prototypes for functions that aren't meant to be used outside of trajectory prediction. usually that would only be a task function and an initialization function. ideally the functions themselves are declared static in the source file as well. I would just move these prototypes to the source file so you don't have to worry about compilation order